recipes/spefile/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.6" %}
+
+package:
+    name: spefile
+    version: {{ version }}
+
+source:
+    path: src
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+
+    run:
+        - python
+        - numpy
+
+test:
+    # Python imports
+    imports:
+        - spefile
+
+about:
+    home: https://pythonhosted.org/pyspec/
```

Comment:
> Is this the same as spefile?

`spefile.py` is a file that came out of the pyspec project.  `pyspec` is a much larger project focused on data analysis of diffraction images.  The file can be seen [here] (https://github.com/stuwilkins/pyspec/blob/master/pyspec/ccd/files.py) in pyspec.  

> pyespec is on PyPI and is GPL-3.0.

You are indeed correct that this should be GPL v3